README.md

```diff
@@ -32,7 +23,27 @@ Your task is to create a React element that represents _you_ (or a close approxi
 </div>
 ```
 
+To start the application type this in the terminal:
+
+```
+npm install
+npm start
+```
+
+To run the test suite type this in the terminal: 
+
+```
+npm test
+```
+
 Feel free to render out the component in the DOM using `ReactDOM.render()`.
 
+## NOTE!! 
```

Comment:
Totally fair — maybe just a little teaser like "don't worry too much about this now — we'll go over it in an upcoming lesson" ?